torch/_refs/nn/functional/__init__.py

```diff
@@ -12,9 +12,44 @@
 from typing import Optional
 
 __all__ = [
+    "celu",
     "elu",
+    "selu",
+    "softplus",
 ]
 
+# celu is implemented specially because it has an alpha argument
+# celu is very similar to elu
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("a",),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.OP_MATH,
```

Comment:
After rebasing this will be KIND.DEFAULT (because OP_MATH was renamed to DEFAULT)